app/models/art_item.rb

```diff
@@ -46,23 +47,38 @@ def has_description
       errors.add(:description, "Can't be blank")
     end
   end
+ 
+  private
 
-  def acceptable_documents
+  def validate_documents
     return unless documents.attached?
-  
-    acceptable_types = [
-      "application/pdf", "text/plain", "image/jpg", 
-      "image/jpeg", "image/png", 
-      "application/vnd.openxmlformats-officedocument.wordprocessingml.document"
-    ]
 
-    documents.each do |doc|
-      unless doc.byte_size <= 20.megabyte
-        errors.add(:documents, "is too big")
+    acceptable_doc_types = ['application/pdf', 'application/msword', 
+                          'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 
+                          'text/plain', 'application/rtf', 'application/vnd.ms-excel', 
+                          'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet']
+
+    documents.each do |document|
+      if document.content_type.in?(acceptable_doc_types)
+        if document.blob.byte_size > 10.megabytes # Adjust the size limit as needed
+          errors.add(:documents, "Document size exceeds the 10MB limit.")
+        end
+      else
+        errors.add(:documents, "Invalid file format. Only documents are allowed.")
       end
+    end
+  end
+
+  def validate_images
+    return unless images.attached?
 
-      unless acceptable_types.include?(doc.content_type)
-        errors.add(:documents, "must be an acceptable file type (pdf,txt,jpg,png,doc)")
+    images.each do |image|
+      if image.blob.content_type.start_with?('image/') # Check if it's an image
+        if image.blob.byte_size > 5.megabytes # Adjust the size limit as needed
+          errors.add(:images, "Image size exceeds the 5MB limit.")
```

Comment:
Please consider mentioning the image file size limit in the front end form.

Reply:
I added the info for size limits